clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/model/KubernetesControllerConverter.groovy

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2017 Cisco, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.clouddriver.kubernetes.model
+
+import io.fabric8.kubernetes.api.model.HasMetadata
+import io.fabric8.kubernetes.api.model.ObjectMeta
+import io.kubernetes.client.models.V1ObjectMeta
+
+class KubernetesControllerConverter implements HasMetadata {
```

Comment:
This is a curious mix between the two API clients, is this the only way?

Reply:
I don't think that this is the only way to address this issue, but it might be less impact the whole code structure.